app/konnect/declarative-config.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Managing a Kong-hosted control plane with declarative config
+no_version: true
+---
+
+Manage entities in your Konnect cloud control plane with declarative
+configuration. To do this, you need [decK](/deck/).
+
+With declarative configuration, you can manage:
+* **All parts of a service:** Service versions, implementations, routes, and
+plugins.
+* **Dev Portal documents:** Specs and markdown files.
+* **Global configuration:** If you want to use global plugins or consumers, you
+have to use declarative configuration. Global plugins appear in the
+Konnect UI as read-only.
```

Comment:
FYI - we are working to enable global plugin support in the UI  in https://konghq.atlassian.net/browse/KHCP-121. This will be released soon (maybe next week...) so you may want to modify this depending on when you plan to publish this content.